packages/core/package.json

```diff
@@ -69,6 +69,7 @@
     "json-schema": "^0.4.0",
     "npm-check-updates": "^15",
     "prettier": "^2.7.1",
+    "projen": "^0.63.4",
```

Comment:
Oh interesting, the ejected projen still has a dependency on itself?

Reply:
Yeah the release tasks reference built-in functions which are not included in the eject. We can kill this dep after setting up changesets.

Reply:
https://github.com/functionless/functionless/actions/runs/3178163655/jobs/5179381959

This seems unrelated to the issue?

Reply:
At first glance, yes, but the run-tasks script the eject process generates is doing dynamic lookup magic. It is trying to find this definition:
https://github.com/projen/projen/blob/main/src/release/bump-version.task.ts